packages/renderer/src/lib/container/ContainerList.spec.ts

```diff
@@ -33,13 +33,18 @@ import { containersInfos } from '../../stores/containers';
 import { providerInfos } from '../../stores/providers';
 import ContainerList from './ContainerList.svelte';
 
-// fake the window.events object
-beforeAll(() => {
+beforeEach(() => {
   vi.mocked(window.listPods).mockResolvedValue([]);
   vi.mocked(window.listViewsContributions).mockResolvedValue([]);
   vi.mocked(window.getContributedMenus).mockResolvedValue([]);
   vi.mocked(window.getConfigurationValue).mockResolvedValue(false);
   vi.mocked(window.onDidUpdateProviderStatus).mockResolvedValue(undefined);
+  vi.mocked(window.listContainers).mockResolvedValue([]);
+  vi.spyOn(window, 'startPod').mockImplementation(vi.fn());
+  vi.spyOn(window, 'startContainer').mockImplementation(vi.fn());
+  vi.spyOn(window, 'removePod').mockImplementation(vi.fn());
+  vi.spyOn(window, 'deleteContainer').mockImplementation(vi.fn());
```

Comment:
```suggestion
beforeEach(() => {
  vi.resetAllMocks();

  vi.mocked(window.listPods).mockResolvedValue([]);
  vi.mocked(window.listViewsContributions).mockResolvedValue([]);
  vi.mocked(window.getContributedMenus).mockResolvedValue([]);
  vi.mocked(window.getConfigurationValue).mockResolvedValue(false);
  vi.mocked(window.onDidUpdateProviderStatus).mockResolvedValue(undefined);
  vi.mocked(window.listContainers).mockResolvedValue([]);
  vi.mocked(window.getProviderInfos).mockResolvedValue([
    {
      name: 'podman',
      status: 'started',
      internalId: 'podman-internal-id',
      containerConnections: [
        {
          name: 'podman-machine-default',
          status: 'started',
        },
      ],
    } as ProviderInfo,
  ]);
```

this fixes all the tests

Reply:
Thanks @axel7083 :)! I haven't checked the [vite.tests.setup.js], now is clear and tests are working.